prime-router/metadata/fhir_mapping/hl7/datatypes/XCN/Practitioner.yml

```diff
@@ -0,0 +1,363 @@
+    type:
+      condition: $xcn13 NOT_NULL
+      expressionType: resource
+      specs: XCN.13
+      valueOf: datatypes/ID/CodeableConcept
+    extension_1-checkDigit:
+      condition: $xcn11 NOT_NULL
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: check-digit
+        valueString:
+          type: STRING
+          expressionType: HL7Spec
+          valueOf: XCN.11
+    extension_2-checkDigitScheme:
+      condition: $xcn12 NOT_NULL
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: check-digit-scheme
+        valueCode:
+          valueOf: XCN.12
+          type: STRING
+          expressionType: HL7Spec
+
+name:
+  expressionType: nested
+  expressionsMap:
+    family:
+      type: STRING
+      valueOf: XCN.2.1
+      expressionType: HL7Spec
+    _family:
+      expressionType: resource
+      valueOf: datatypes/FN/ExtensionHumanName-family
+      specs: XCN.2
+    given:
+      generateList: true
+      expressionType: nested
+      expressions:
+        - expressionType: HL7Spec
+          valueOf: XCN.3
+          type: STRING
+        - expressionType: HL7Spec
+          valueOf: XCN.4
+          type: STRING
+    suffix:
+      generateList: true
+      expressionType: nested
+      expressions:
+        - expressionType: HL7Spec
+          valueOf: XCN.5
+          type: STRING
+        - expressionType: HL7Spec
+          valueOf: XCN.7
+          type: STRING
+        - expressionType: HL7Spec
+          valueOf: XCN.21
+          type: STRING
+    prefix:
+      type: STRING
+      valueOf: XCN.6
+      expressionType: HL7Spec
+    use:
+      type: NAME_USE
+      valueOf: XCN.10
+      expressionType: HL7Spec
+    # This needs to use variables because the message parses this field using NULLDT since it is deprecated which means that
+    # the values do not get correctly loaded and will not be mapped over correctly using a spec.
+    period_1:
+      vars:
+        xcn19: String, XCN.19
+        xcn20: String, XCN.20
+        start: String, XCN.17.1
+        end: String, XCN.17.2
+      condition: $xcn19 NULL && $xcn20 NULL
+      expressionType: resource
+      valueOf: datatypes/DR/Period
+    # XCN.17 is deprecated so the schema favors using XCN.19/20 if either are populated
+    period_2:
+      vars:
+        xcn19: String, XCN.19
+        xcn20: String, XCN.20
+      condition: $xcn19 NOT_NULL || $xcn20 NOT_NULL
+      expressionType: nested
+      expressionsMap:
+        start:
+          valueOf: "GeneralUtils.dateTimeWithZoneId(periodStart,ZONEID)"
+          expressionType: JEXL
+          vars:
+            periodStart: XCN.19
+        end:
+          valueOf: "GeneralUtils.dateTimeWithZoneId(periodEnd,ZONEID)"
+          expressionType: JEXL
+          vars:
+            periodEnd: XCN.20
+    extension_AssemblyOrder:
```

Comment:
The mapping inventory placed this extension within `name.family`, but per the extension definition, `name` is the correct location.